lib/galaxy/managers/executables.py

```diff
@@ -1,6 +1,6 @@
 """Utilities for loading tools and workflows from paths for admin user requests."""
 
-from gxformat2.converter import ordered_load
+import yaml
```

Comment:
I'm certain I had this in here for a reason. Is it just not needed on Python 3?

Reply:
Yep, on python 3.6 dicts became ordered, and since 3.7 that behavior is guranteed